config/project/navigation/navs/fineprintNavigation--ef1fcb64-1148-4086-92c8-fb30518b08f2.yaml

```diff
@@ -0,0 +1,102 @@
+defaultPlacement: end
+fieldLayouts:
+  0d28b0c3-f456-411f-9dcc-793ca304f681:
+    tabs:
+      -
+        elementCondition: null
+        elements:
+          -
+            autocapitalize: true
+            autocomplete: false
+            autocorrect: true
+            class: null
+            dateAdded: '2025-04-30T03:55:36+00:00'
+            disabled: false
+            elementCondition: null
+            id: null
+            includeInCards: false
+            inputType: null
+            instructions: null
+            label: null
+            max: null
+            min: null
+            name: null
+            orientation: null
+            placeholder: null
+            providesThumbs: false
+            readonly: false
+            requirable: false
+            size: null
+            step: null
+            tip: null
+            title: null
+            type: craft\fieldlayoutelements\TitleField
+            uid: 31a976ab-c7db-4497-b749-ca556fb577d1
+            userCondition: null
+            warning: null
+            width: 100
+          -
+            dateAdded: '2025-04-30T03:55:36+00:00'
+            elementCondition: null
+            includeInCards: false
+            instructions: null
+            label: null
+            providesThumbs: false
+            required: false
+            tip: null
+            type: verbb\navigation\fieldlayoutelements\NodeTypeElements
+            uid: a10e9f14-8ab5-4b0b-90a6-8e0013ce72a8
+            userCondition: null
+            warning: null
+        name: Node
+        uid: c9f5a894-8712-43a5-8195-1bfd6d47d38d
+        userCondition: null
+handle: fineprintNavigation
+instructions: ''
```

Comment:
Would be helpful to indicate to authors _where_ the navigation will be rendered, as well as any limits on number of links / nav items. Same for all the three navs.